spec/support/driver_setting.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.configure do |config|
+	config.before(:each, type: :system) do
+		# driven_by(:rack_test)
+		# driven_by(:selenium_chrome)
+		driven_by(:selenium_chrome_headless)
+	end
+end
```

Comment:
インデントがタブになってしまっているので、Rubyの開発ではスペース2でお願いします！
```suggestion
RSpec.configure do |config|
  config.before(:each, type: :system) do
    # driven_by(:rack_test)
    # driven_by(:selenium_chrome)
    driven_by(:selenium_chrome_headless)
  end
end
```

Reply:
インデントをスペースに変更しただけだとgitに変更したと判断されなかったので、これだけをコミットすることができませんでした。
次のコミットで反映されていると思います。